src/machine/asm/execute_aarch64.S

```diff
@@ -336,6 +339,9 @@ ckb_vm_x64_execute:
   add INST_PC, TRACE, CKB_VM_ASM_TRACE_OFFSET_THREAD
   NEXT_INST
 .prepare_trace:
+  ldrb TEMP2w, [PAUSE]
```

Comment:
I wasn't sure on this, can you check the literature if we need atomic loads on aarch64?

Reply:
I compiled `Pause`'s `has_interpreted` function on aarch64, the result is as follows:

```
0000000000000000 <_ZN2rs5Pause15has_interrupted17h49eea8a15effbf29E>:
   0:   f9400008        ldr     x8, [x0]
   4:   91004108        add     x8, x8, #0x10
   8:   08dffd08        ldarb   w8, [x8]
   c:   7100011f        cmp     w8, #0x0
  10:   1a9f07e0        cset    w0, ne  // ne = any
  14:   d65f03c0        ret
```

Looks like we need to use `ldarb`.

In addition, on x64, the result is 

```
0000000000000000 <_ZN2rs5Pause15has_interrupted17h64f564067071b9e2E>:
   0:   48 8b 07                mov    (%rdi),%rax
   3:   0f b6 40 10             movzbl 0x10(%rax),%eax
   7:   84 c0                   test   %al,%al
   9:   0f 95 c0                setne  %al
   c:   c3                      ret
```

Reply:
Yeah I think that's because on x64, `movzbl` guarentees atomic load, but on `aarch64` we need to explicit use the atomic load operations

Reply:
`Ordering::Relaxed` essentially means "I don't care in what order you load the memory, it could happen before the other side writes the stop signal, or after that", here we need acquiring load to ensure that loading operations happen after any flying storing operations. I do believe we need `ldarb` here. In any case the compiler will not warn us, this will just be a mysterious runtime error later when at certain chip / OS configuration, we found that pausing does not work.

Reply:
Typically when I write multi-threaded software, I use `release` ordering store and `acquire` ordering load, that can get you correct for the majority of the time. Only in very rare situations we would need `acq-rel` or even `seq-cst`.